src/library_idbfs.js

```diff
@@ -280,6 +280,12 @@ addToLibrary({
         }
       };
 
+      // transaction may abort if (for example) there is a QuotaExceededError
+      transaction.onabort = (e) => {
+        done(e.target.error); // DOMException
+        e.preventDefault();
```

Comment:
Maybe we could just `transaction.onerror = transaction.onabort = (e) => {`.. above?

Reply:
Did some quick testing. Looks like `this.error` is undefined when onabort is triggered, so that doesn't work. (Looks like we really need to get it from the event - in fact, I'm not sure whether `onerror` should be doing this as well, but I haven't yet come across a setup to trigger `onerror`.)

Reply:
Using `this` in the onerror callback looks wrong anyway.. since that function is an arrow function `this` will just be the global object I think.   Perhaps when we converted `onerro` from a function to an arrow function this was overlooked?

Reply:
> Using `this` in the onerror callback looks wrong anyway.. since that function is an arrow function `this` will just be the global object I think. Perhaps when we converted `onerro` from a function to an arrow function this was overlooked?

Yep, looks to be the case. Pushed a commit to use `e.target.error` in all cases.